app/models/schedule_rule.rb

```diff
@@ -31,8 +31,8 @@ def self.unavailable_for_date(instrument, day)
     rules.each do |rule|
       res = Reservation.new({
         :product => instrument,
-        :reserve_start_at => day.dup.change(:hour => rule.start_hour, :min => rule.start_min),
-        :reserve_end_at => day.dup.change(:hour => rule.end_hour, :min => rule.end_min),
+        :reserve_start_at => day.to_time.change(:hour => rule.start_hour, :min => rule.start_min),
```

Comment:
I really wanted to update this method, but I like that looking at this diff emphasizes the change is just from `dup` to `to_time`. Now that we have some tests on this method, I'm more comfortable with bigger changes.